docs/gettingstarted/installation.rst

```diff
@@ -263,19 +263,18 @@ You'll probably want to setup at least two nodes.
    At the time of writing, the only supported Fedora 20 kernel on DigitalOcean is ``Fedora 20 x64 vmlinuz-3.16.6-203.fc20.x86_64``.
    To switch to that kernel, follow these steps:
 
-   #. Upgrade the kernel package inside the virtual machine:
-
-      This specific kernel is no-longer available from the standard Fedora 20 repositories, so we install from the ``kojipkgs`` repository directly.
-
-      .. code-block:: sh
-
-         [root@digitalocean]# yum update https://kojipkgs.fedoraproject.org/packages/kernel/3.16.6/203.fc20/x86_64/kernel-3.16.6-203.fc20.x86_64.rpm
-
    #. Configure the Droplet to boot with the desired kernel:
 
       * Go to the DigitalOcean control panel for your specific Droplet, and in the Settings section choose the Kernel tab.
       * Choose the ``Fedora 20 x64 vmlinuz-3.16.6-203.fc20.x86_64`` kernel for Fedora 20 (scroll all the way to the bottom) and press "Change".
 
+   #. Upgrade the kernel package inside the virtual machine:
+
+      This specific kernel is no-longer available from the standard Fedora 20 repositories, so we install from the ``kojipkgs`` repository directly.
```

Comment:
I'd just say "from koij". It isn't a yum repository, and the service is called koji.